integration/test_overload_repro.py

```
@@ -0,0 +1,276 @@
+        failures = 0
+        timeouts = 0
+        while not stop_event.is_set():
+            try:
+                result = client.execute_command("FT.SEARCH", "idx", query, "LIMIT", "0", "10")
+                successes += 1
+            except ResponseError as e:
+                if "timeout" in str(e).lower() or "overloaded" in str(e).lower():
+                    timeouts += 1
+                else:
+                    failures += 1
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)